services/accounts/accounts.go

```diff
@@ -221,7 +301,7 @@ func (api *API) verifyPassword(password string) error {
 	return err
 }
 
-func (api *API) addAccountWithMnemonic(
+func (api *API) addAccountWithMnemonicPasswordVerified(
```

Comment:
Does the name `addAccountWithMnemonicPasswordVerified` mean that the password has already been verified?

If so, perhaps you could add some go docs to explain the function. Something like:

```suggestion
// addAccountWithMnemonicPasswordVerified adds an accounts.Account derived from the given Mnemonic ... (some info about the usage of the password)
func (api *API) addAccountWithMnemonicPasswordVerified(
```

Reply:
@Samyoul thanks, yes, will do that.